src/components/message/styles.scss

```diff
@@ -172,13 +172,10 @@
 
   &__author-avatar {
     visibility: hidden;
-    background-position: center;
-    background-repeat: no-repeat;
-    background-size: cover;
-    width: 32px;
-    height: 32px;
-    overflow: hidden;
-    border-radius: 50%;
+
+    > div:first-child {
```

Comment:
This is pretty dicey. We should never rely on the internal structure of a component library. How 'bout `> *` which is any direct descendant.

Reply:
ah I should have gone with my instinct, I used `> *` before changing to `> div`. Shall update before merging. Thanks Dale !